### ISSUE TITLE
report  command occurred abort (core dumped) 

### ISSUE THREAD

**Issue by yanxueqing621** [hahnlab/CAFE#25]
```
#!cafe
load -i filtered_cafe_input.txt -t 10 -l cafe.log
tree (((Nym:185.832403,(((((AT:89.277535,Potri:89.277535):11.398834,(Solyc:91.410783,Bv:91.410783):9.265586):5.535909,VIT:106.212277):18.787723,NNU:125.000000)Eudicots:41.420759,((GSMUA:130.230110,(Aco:116.317945,(Sobic:65.000000,LOC_Os:65.000000)Grass:51.317945):13.912164):19.123848,(Spipo:136.145436,Zosma:136.145436):13.208521)Monocots:17.066802):19.411644):14.167597,scaffold:200.000000)Angio:148.774455,Gb_:348.774455)
lambda -s -t (((1,(((((1,1)1,(1,1)1)1,1)1,1)1,((1,(1,(1,1)1)1)1,(1,1)1)1)1)1,1)1,1)
report cafe.report
```
[filtered_cafe_input.txt](https://github.com/hahnlab/CAFE/files/1317295/filtered_cafe_input.txt)

load , tree and lambda command can run normally, but report command will occurred "core dumped" error.

**Comment by yanxueqing621** [comment]
when I change parameter '-t' to 1,  report can run normally.

**Comment by benfulton** [comment]
Hi, and thanks for the report. I'm looking into it.

**Comment by benfulton** [comment]
A threading issue almost certainly. We will remove the threading around the Viterbi calculation until it can be perfected. This issue will be fixed with commit b61e5eb776feba92b5bb8e749b73c840586ae0d8
